libbeat/docs/field-name-changes.asciidoc

```diff
@@ -2,6 +2,7 @@
 [frame="topbot",options="header"]
 |======================
 |Old Field|New Field
+|`auditd.message_type`            |`event.type`
```

Comment:
@webmat Do we need to revert this change?

Reply:
Yes it should :-)